tests/e2e/docker-compose.yml

```diff
@@ -105,7 +107,7 @@ services:
     profiles:
       - synpress
     container_name: agoric_chain
-    image: ghcr.io/agoric/agoric-3-proposals:main
+    image: ghcr.io/agoric/agoric-3-proposals:${A3P_BRANCH}
```

Comment:
this is a Docker tag, not branch.

what's publishing an 'ollinet' tag? I don't see one in https://github.com/Agoric/agoric-3-proposals/pkgs/container/agoric-3-proposals

Reply:
this docker tag can only be used for local testing and not for ollinet.
for specific a3p tag on ollinet, I will have to make change in instagoric that I can do separately. 
removing this change for now.

Reply:
@turadg updated the PR.